pytest.ini

```diff
@@ -0,0 +1,31 @@
+[pytest]
```

Comment:
Do we still want to move the pytest config into its own `pytest.ini` file? Asking because I'm not sure whether this was done as an attempt to fix the `filename` attributes or if it was done for tidying purposes. No strong preference on my side. Thoughts?

Reply:
Yes, to be honest for tidying purposes, but I would not know if reverting it would be safe, would need to test it. And it is a change I would do anyway, so I would prefer to keep it, but I could also put it into a tidying PR (e.g. https://github.com/aconrad/pycobertura/pull/141/files).